server/sonar-server/src/main/java/org/sonar/server/es/BaseIndexer.java

```diff
@@ -58,14 +54,14 @@ protected BaseIndexer(EsClient client, long threadKeepAliveSeconds, String index
       threadKeepAliveSeconds, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>());
   }
 
-  public void index() {
+  public void index(final IndexTask task) {
```

Comment:
rename IndexerTask ? to be clear it's not related to SourceLineIndex for example